__tests__/app.test.js

```diff
@@ -127,4 +127,49 @@ describe("GET /api/articles", () => {
         expect(body.msg).toBe("Route not found");
       });
   });
+  test("GET:404 sends an appropriate status and error meaasge when given a valid but non existent id", () => {
+    return request(app)
+      .get("/api/articles/123")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("article doesn't exist");
+      });
+  });
+});
+
+describe.only("GET /api/articles/:article_id/comments", () => {
```

Comment:
make sure to remove .only when you push or future prs wont get reviewed as we cant be sure that all of your tests pass at the time of committing